### PR TITLE
add tooltip warning when recipient address is the same as sender address

### PR DESCRIPTION
**Pull request by yjkimjunior**
addresses #252

### CODE REVIEW

**Comment by amaurymartiny** [comment]
Is it possible to not disable the Send button, but just show the tooltip?

Esp on testnets, I often send stuff to myself.

**Comment by Tbaut** [comment]
Thanks a lot for the contribution. I get an empty tooltip when I just click in the "amount" field:
image

Same above the slider:
image

Also if we ask: "are you sure you want to send.." as a user I feel like I can answer and make that tooltip go away, but I can't and it stays here forever... which I don't find user friendly.

Could we add a way to close it and reformulate it so that it's not a question:
"Warning: You are sending this to yourself" or "Warning: the sender and receiver address are the same"

**Comment by Tbaut** [comment]
that's cool.
- could we put the warning above the send button?
- I keep breaking it sorry, if you enter your own address (it gives the warning) and then you remove it (because you want to correct it) the tooltip is offset due to the old offset from the warning.
Should be:
image
but is:
image

File: packages/fether-react/src/assets/sass/shared/_form.scss [hunk 1]
```diff
@@ -110,6 +110,8 @@ input.form_field_amount[type='number'] {
   font-size: ms(6);
   line-height: ms(6) * 1.1;
   font-weight: 200;
+  padding: 3rem 0;
```
Comment (Tbaut) [inline packages/fether-react/src/assets/sass/shared/_form.scss]: Any reason for making the amount field so big?

Comment (yjkimjunior) [reply packages/fether-react/src/assets/sass/shared/_form.scss]: I was playing around with it while trying to address the tooltip blocking issue, and adding some padding made the screen felt a bit less suffocating. So was not much reasoning but more of a feeling :)

Comment (Tbaut) [reply packages/fether-react/src/assets/sass/shared/_form.scss]: From the few user interviews I've had so far, they really liked our design. I'd keep it this way (without the margin and padding) for now. We might need to add some space if we add some more info (like the amount in $), but that's the future :)

File: packages/fether-react/src/Send/TxForm/TxForm.js [hunk 2]
```diff
@@ -106,6 +112,14 @@ class Send extends Component {
                             step={0.5}
                             type='range' // In Gwei
                           />
+                          {values.to === values.from ? (
```
Comment (amaurymartiny) [inline packages/fether-react/src/Send/TxForm/TxForm.js]: Can be simplified into `values.to === values.from && ...`

File: packages/fether-react/src/Send/TxForm/TxForm.js [hunk 3]
```diff
@@ -68,7 +68,13 @@ class Send extends Component {
                     initialValues={{ from: accountAddress, gasPrice: 4, ...tx }}
                     onSubmit={this.handleSubmit}
                     validate={this.validateForm}
-                    render={({ handleSubmit, valid, validating, values }) => (
+                    render={({
+                      handleSubmit,
+                      form,
```
Comment (amaurymartiny) [inline packages/fether-react/src/Send/TxForm/TxForm.js]: Is this variable used somewhere?

Comment (yjkimjunior) [reply packages/fether-react/src/Send/TxForm/TxForm.js]: ah that was an artifact from the warning engine. removed it in the next commit.

**Pull request merged**